### PR TITLE
feature (ref dplan-11397): define demosplanKernel as a type of variable

### PR DESCRIPTION
**Pull request by jalalifardemos**
**Ticket:** https://demoseurope.youtrack.cloud/issue/DPLAN-11397/Phpstan-level-2-Error-beheben
**Fehler**: 
```                          
 ------ ------------------------------------------------------------------------------------------ 
Call to an undefined method Symfony\Component\HttpKernel\KernelInterface::getActiveProject().
 ------ ------------------------------------------------------------------------------------------ 

```

### How to review/test
`bin/{projektName} dp:phpstan -l 2`

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php [hunk 1]
```diff
@@ -41,8 +42,9 @@ public function doRun(
     private function addProjectFolderConsoleDeprecationNotice(OutputInterface $output): void
     {
         if ($this->isDeprecatedFrontController) {
-            $activeProject = $this->getKernel()->getActiveProject();
-
+            $kernel = $this->getKernel();
+            /** @var DemosPlanKernel $kernel */
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php:46]: In general I would set this comment at the place where the variable is defined (one line above) but no blocker

File: demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php [hunk 2]
```diff
@@ -22,6 +22,7 @@ public function __construct(KernelInterface $kernel, private readonly bool $isDe
         parent::__construct($kernel);
 
         /* @var DemosPlanKernel $kernel */
+        // @phpstan-ignore-next-line
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php:25]: wouldn't it be enough to write `/**` the line above?

**Review by mussbach — CHANGES_REQUESTED** [review]
thx. I would set the comment a bit differently, but no blocker. phpstan-ignore-next-line could be avoided imho

File: demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php (hunk 2 above)
Comment (jalalifardemos) [reply demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php:25]: I don't know why CS fixer do it

Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php:25]: really? uh, that is interesting

Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php:25]: but the cs fixer did not touch the line above, if I am not wrong

Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php:25]: ok, weird @shumyachkin could you have a look pls?

Comment (mussbach) [reply demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php:25]: @jalalifardemos I modified the phpcs config. Could you please try again to use /** comments?

Comment (jalalifardemos) [reply demosplan/DemosPlanCoreBundle/Application/ConsoleApplication.php:25]: done in [5d5dc55](https://github.com/demos-europe/demosplan-core/pull/2930/commits/5d5dc558ea1a77f390a1230d1fade12ed4520d71) but CS-Fixer change it again in [e1a3db9](https://github.com/demos-europe/demosplan-core/pull/2930/commits/e1a3db936ba761d18247250e6b855203a9736d2a)

**Review by mussbach — APPROVED** [review]
lgtm, thx but please change the branch to main

**Pull request merged**